share/completions/az.fish

```diff
@@ -0,0 +1,16 @@
+function __fish_az_complete
+  set -x _ARGCOMPLETE 1
```

Comment:
All of these should explicitly be set in local scope - `set -lx`. Otherwise you'd clobber any global (or worse, universal) variable of that name.

Reply:
Done